### ISSUE TITLE
Unhandled UI exception when opening materials shopping list panel in Grid.

### ISSUE THREAD

**Issue by Gimi1967** [EDDiscovery/EDDiscovery#1578]
### Description
I was adding, resizing and deleting panels in the grid view. The exception came when I tried to add the materials shopping list. Scan data and engineer planner already displaying on the grid.

### Additional Information

[Trace_20171228010034.0.txt](https://github.com/EDDiscovery/EDDiscovery/files/1590088/Trace_20171228010034.0.txt)

### Exception Details:
>There was an unhandled UI exception.
>```
>==== BEGIN ====
>System.InvalidOperationException: SplitterDistance must be between Panel1MinSize and Width - Panel2MinSize.
>   at System.Windows.Forms.SplitContainer.set_SplitterDistance(Int32 value)
>   at System.Windows.Forms.SplitContainer.ApplyPanel1MinSize(Int32 value)
>   at System.Windows.Forms.SplitContainer.set_Panel1MinSize(Int32 value)
>   at EDDiscovery.UserControls.UserControlShoppingList.Display() in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlShoppingList.cs:line 264
>   at EDDiscovery.UserControls.UserControlSynthesis.Display() in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlSynthesis.cs:line 276
>   at EDDiscovery.UserControls.UserControlShoppingList.InitialDisplay() in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlShoppingList.cs:line 100
>   at EDDiscovery.UserControls.UserControlContainerGrid.CreatePanel(UserControlCommonBase uccb, Point pos, Size size) in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlContainerGrid.cs:line 192
>   at EDDiscovery.UserControls.UserControlContainerGrid.<buttonExtPopOut_Click>b__25_1(Object s, EventArgs ea) in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlContainerGrid.cs:line 297
>   at ExtendedControls.DropDownCustom._listcontrol_SelectedIndexChanged(Object sender, EventArgs e) in C:\Code\EDDiscovery\ExtendedControls\Controls\DropDownCustom.cs:line 69
>   at ExtendedControls.ListControlCustom.OnMouseClick(MouseEventArgs e) in C:\Code\EDDiscovery\ExtendedControls\Controls\ListControlCustom.cs:line 311
>   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
>   at System.Windows.Forms.Control.WndProc(Message& m)
>   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
>===== END =====
>```

**Comment by iainross** [comment]
I've had a few goes and I can't reproduce this - have you got a reliable set of reproduction steps?

**Comment by Gimi1967** [comment]
I have just tried this on a second computer and I am unable to recreate the problem as well. It does persist on my main installation of EDD though. 

Some more details.
I run EDD on a separate computer accessing the journals over LAN.
I use EDD portable 9.0.1 on Windows 10 professional.
If I click continue after the exception occurs the panel is open, but in the background behind all other panels. The size and layout of the panel is messed up when it opens.
-> Of note: The messed up layout happens on both computers but it's fixable by resizing the window..

I'm wondering if one of my setup files is messed up. I have a complete backup of my 8.4.2 installation and the problem is present there as well.
It's worth noting that when I upgrade my installation I just delete the program files and copy over the new version, leaving the entire data directory intact.
It may well be that some maintenance is due on my EDD data.

->  And I just found out that it's theme related. I have customised the fonts and colours on my EDD installation. Reverting to the "Windows Default" theme fixes the issue.
Using the theme file in question recreates the issue on both my computers I have included the theme file here.

[Gimi-has-issue.txt](https://github.com/EDDiscovery/EDDiscovery/files/1595178/Gimi-has-issue.txt)
